### DISCUSSION TITLE
無限奈落の完成度

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#89] [エリア・建築の完成度・開発状況]
- [x] 現行の建築鯖に移行中
_鯖環境が1.13.2からが1.16.4になったついでに設置予定。_

**Comment by mocope** [comment]
通常有限資源となっているものを無限奈落の商人等に売らせる
ランクメモリスターやバッチ(https://github.com/TUSB/TheUnusualSkyBlock/issues/297 )、場合によってはケイオスコア(https://github.com/TUSB/TheUnusualSkyBlock/issues/267 )など

バランスを壊しかねない存在も無限奈落に追放しても良いかも(言ってしまえば裏株)
